src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetInstaller.cs

```diff
@@ -214,13 +220,14 @@ public async Task<InstallResult> InstallAsync(InstallRequest installRequest, IMa
                 {
                     Author = nuGetPackageInfo.Author,
                     Owners = nuGetPackageInfo.Owners,
+                    Vulnerabilities = sb.ToString(),
```

Comment:
we don't need to store vulnerabilities. They should be always checked using API.